functions/common/geode-common/src/main/java/org/springframework/cloud/fn/common/geode/GeodeSecurityProperties.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2017-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.fn.common.geode;
+
+import java.util.Properties;
+
+import org.apache.geode.LogWriter;
+import org.apache.geode.distributed.DistributedMember;
+import org.apache.geode.security.AuthInitialize;
+import org.apache.geode.security.AuthenticationFailedException;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author David Turanski
+ **/
+
+@ConfigurationProperties("geode.security")
+public class GeodeSecurityProperties {
+
+	/**
+	 * The cache username.
+	 */
+	private String username;
+
+	/**
+	 * The cache password.
+	 */
+	private String password;
+
+	public String getUsername() {
+		return username;
+	}
+
+	public void setUsername(String username) {
+		this.username = username;
+	}
+
+	public String getPassword() {
+		return password;
+	}
+
+	public void setPassword(String password) {
+		this.password = password;
+	}
+
+	@SuppressWarnings("unused")
+	public static class UserAuthInitialize implements AuthInitialize {
+
+		private static final String USERNAME = "security-username";
+		private static final String PASSWORD = "security-password";
+
+		private LogWriter securitylog;
+		private LogWriter systemlog;
```

Comment:
Shows as deprecated in the IDE.

Reply:
The docs don't say what the replacement is. Something about use Log4j2 but no examples that I can find. @jxblum ?

Reply:
AFAICT this can't be fixed since the AuthInitialize.init signature still requires LogWriter